src/main/java/wooteco/subway/line/dto/LineRequest.java

```diff
@@ -1,21 +1,39 @@
 package wooteco.subway.line.dto;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Positive;
+
 public class LineRequest {
+    @NotBlank(message = "name은 필수로 입력하여야 합니다.")
+    @Pattern(regexp = "^[가-힣0-9]{2,10}$", message = "노선 이름은 2~20자 이하의 한글/숫자만 가능합니다")
     private String name;
+    @NotBlank(message = "Color은 필수로 입력하여야 합니다.")
     private String color;
```

Comment:
```suggestion
    @NotBlank(message = "name은 필수로 입력하여야 합니다.")
    @Pattern(regexp = "^[가-힣0-9]{2,10}$", message = "노선 이름은 2~20자 이하의 한글/숫자만 가능합니다")
    private String name;
    
    @NotBlank(message = "Color은 필수로 입력하여야 합니다.")
    private String color;
```

공백 라인을 두는 게 좋을 것 같아요!

Reply:
반영했습니다 감사합니다!! ㅎㅎ